src/app/problems/reducers/index.ts

```diff
@@ -20,11 +23,24 @@ export const getProblemsEntitiesState = createSelector(
     state => state.problems
 );
 
-export const getProblems = createSelector(
+export const getProblemsEntitiesArray = createSelector(
     getProblemsEntitiesState,
     state => state.entities
 );
 
+export const getProblems = createSelector<any, Dictionary<Problem>, Dictionary<Problem>>(
+    getProblemsEntitiesArray,
+    (problemEntities) => {
+        if (!problemEntities) { return {} }
+
+        Object.keys(problemEntities).map((key) => {
+            problemEntities[key].createdAt = moment(problemEntities[key].createdAt);
```

Comment:
Please make sure this operation does not override the State content, but only modifies local variable.